user-stories.md

```diff
@@ -9,11 +8,8 @@
 
 3. I should be able to search by specific statistic and filter through for each one based on the numbers that I input.
 
-4. There should be a separate input that I can search multiple countries to have them live on the page and compare them. 
+4. There should be a separate input that I can search multiple countries to have them live on the page and compare them.
 
 5. As a user I should be able to star a country and have it saved to local storage. I should be able to view my starred countries on page reload. I should be able to unstar my favorite countries so they don't persist on page reload.
 
-6. 
-
--add flags to each country
--
```

Comment:
I think we should delete this file all from our directory.

Reply:
Agreed

Reply:
Will do!